packages/data-store/src/types/contact/contact.ts

```diff
@@ -150,9 +150,15 @@ export type Contact = NaturalPerson | Organization
 export type NonPersistedContact = NonPersistedNaturalPerson | NonPersistedOrganization
 export type PartialContact = PartialNaturalPerson | PartialOrganization
 
+export enum PartyOrigin {
+  INTERNAL = 'internal',
+  EXTERNAL = 'external',
+}
+
 export type PartyType = {
   id: string
   type: PartyTypeEnum
+  origin?: PartyOrigin
```

Comment:
We discussed in the sprint start to make them mandatory with a default of external

Reply:
Made origin mandatory in PartyType